strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/layout/maven/MavenMetadataManagementControllerTest.java

```diff
@@ -60,27 +50,19 @@ public void init()
     {
         super.init();
 
-        // Create repositories
-        MutableRepository repositoryReleases = mavenRepositoryFactory.createRepository(REPOSITORY_RELEASES);
-        repositoryReleases.setPolicy(RepositoryPolicyEnum.RELEASE.getPolicy());
-
-        createRepository(STORAGE0, repositoryReleases);
-
-        MutableRepository repositorySnapshots = mavenRepositoryFactory.createRepository(REPOSITORY_SNAPSHOTS);
-        repositorySnapshots.setPolicy(RepositoryPolicyEnum.SNAPSHOT.getPolicy());
-
-        createRepository(STORAGE0, repositorySnapshots);
+    }
 
-        // Generate artifacts
-        generateArtifact(getRepositoryBasedir(STORAGE0, REPOSITORY_RELEASES).getAbsolutePath(),
-                         "org.carlspring.strongbox.metadata:strongbox-metadata",
-                         new String[]{ "3.0.1",
-                                       "3.0.2",
-                                       "3.1",
-                                       "3.2" });
+    @ExtendWith({ RepositoryManagementTestExecutionListener.class})
+    @Test
+    public void testRebuildSnapshotMetadata(@MavenRepository(repositoryId = REPOSITORY_SNAPSHOTS, layout = MavenArtifactCoordinates.LAYOUT_NAME, policy = RepositoryPolicyEnum.SNAPSHOT) Repository repository)
```

Comment:
You can also drop the `layout`, because the `MavenRepository`'s default value for this is already `String layout() default Maven2LayoutProvider.ALIAS;`.